packages/account/src/Components/trading-assessment/__tests__/test-warning-modal.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { isDesktop, isMobile } from '@deriv/shared';
+import TestWarningModal from '../test-warning-modal';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(),
+    isDesktop: jest.fn(),
+}));
+
+describe('<TestWarningModal />', () => {
+    let modal_root_el: HTMLDivElement;
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('shoukd render modal correctly on desktop', () => {
+        (isDesktop as jest.Mock).mockReturnValue(true);
+        const props = {
+            show_risk_modal: true,
+            title: 'Appropriateness Test Warning',
+            body_content: <div>body_content</div>,
+            footer_content: <div>fotter_content</div>,
+        };
+        render(<TestWarningModal {...props} />);
+
+        const text = screen.getByText('Appropriateness Test Warning');
+
+        expect(text).toBeInTheDocument();
+    });
+
+    it('shoukd render modal correctly on mobile', () => {
+        (isMobile as jest.Mock).mockReturnValue(true);
+        (isDesktop as jest.Mock).mockReturnValue(false);
+        const props = {
+            show_risk_modal: true,
+            title: 'Appropriateness Test Warning',
+            body_content: <div>body_content</div>,
+            footer_content: <div>fotter_content</div>,
+        };
```

Comment:
maybe we could move this const props out and share it between the 2 test cases

Reply:
is this done @sanjam-deriv